### ISSUE TITLE
Can't change prop fill after icon mounted.

### ISSUE THREAD

**Issue by leyenda** [antonreshetov/vue-unicons#47]
Hello everyone,

The prop fill is assigned to a data variable called localFill, only when the component is being created. 

```js
data () {
    return {
      localFill: this.fill
    }
  }
``` 
That makes not posible to change the fill prop once the icon is mounted.
You can see in this [example](https://codesandbox.io/s/vue-template-forked-pbu4n?file=/src/App.vue).

Maybe setting a watcher on the prop fill that update localFill is the fix?

**Comment by antonreshetov** [comment]
Closed because it was fixed in #48

**Comment by leyenda** [comment]
Thank you for the fix, may I ask if it can be released?
